src/main/java/com/to/ordermanagementservice/entity/Electronic.java

```diff
@@ -0,0 +1,36 @@
+package com.to.ordermanagementservice.entity;
+
+import java.math.BigDecimal;
+import java.time.OffsetDateTime;
+
+public class Electronic extends Product {
+    private String brand;
+    private OffsetDateTime warrantyPeriod;
```

Comment:
What else classes from Java Date Time API you can use for the  'warrantyPeriod'?